api/src/main/java/com/epam/esm/handler/RuntimeExceptionHandler.java

```diff
@@ -33,6 +30,21 @@
         return ResponseEntity.status(httpStatus).body(exceptionResponseBody);
     }
 
+    /**
+     * Handle InvalidFieldValueException.
+     *
+     * @param e the exception that handler handle.
+     * @return the response entity
+     */
+    @ExceptionHandler(InvalidURLParameterException.class)
```

Comment:
Again, you don't have a global exception handler that handles exceptions not caught by other handlers.
For this, you should add `@ExceptionHandler(Exception.class)`

Reply:
Create hadler